

<launch>
    <rosparam 
        command="load" 
        file="$(find zumo)/config/param.yaml" 
    />
<!--
    <node pkg="rosserial_python" name="serial_node" type="serial_node.py">
        <param name="port" value="/dev/ttyACM0"/>
        <param name="baud" value="9600"/>
    </node>
-->
    <node 
        pkg="zumo" 
        name="zumo_pi" 
        type="zumo_pi.py"
        output="screen"
    />  
</launch>
